### PR TITLE
I never want to see a .spec file again in my life.

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Proposed Changes

Updates a bunch of plugins for the hub launch

### CODE REVIEW

File: pastebin/help.md [hunk 1]
```diff
@@ -6,16 +6,17 @@ sources code or configuration information, but anyone is more than welcome to
...
+The Pastebin plugin allows you to post text online, and analyze posts. 
+
 # Key Features
 
-* Feature 1
-* Feature 2
-* Feature 3
+* Send text to Pastebin
+* Trigger workflows on new posts
+* Get ana analyze posts
```
Comment (mhofert-r7) [inline pastebin/help.md]:
```suggestion
* Get and analyze posts
```

**Comment by jherlitz-r7** [comment]
Nice commit name

File: pastebin/help.md (hunk 1 above)
Comment (hwilsey-r7) [inline pastebin/help.md]: ana -> and

File: pdf_generator/help.md [hunk 2]
```diff
@@ -1,17 +1,14 @@
 # Description
 
-The PDF Generator plugin creates a PDF from user provided data.
+The PDF Generator plugin creates a PDF from user provided data. This allow the user to create robust reports to distribute workflow details.
```
Comment (hwilsey-r7) [inline pdf_generator/help.md]: allow -> allows

File: pdf_generator/plugin.spec.yaml [hunk 3]
```diff
@@ -3,21 +3,20 @@ extension: plugin
 product: ["insightconnect"]
 name: pdf_generator
 title: PDF Generator
-description: The Portable Document Format (PDF) is a file format developed by Adobe
-  in the 1990s to present documents, including text formatting and images, in a manner
-  independent of application software, hardware, and operating systems
+description: The PDF plugin allows the user to create PDF reports from workflow data.
```
Comment (lmilby-r7) [inline pdf_generator/plugin.spec.yaml]:
```suggestion
description: The PDF plugin allows the user to create PDF reports from workflow data
```

File: ping/plugin.spec.yaml [hunk 4]
```diff
@@ -3,19 +3,20 @@ extension: plugin
 product: ["insightconnect"]
 name: ping
 title: Ping
-description: Ping networking tool
+description: The Ping plugin is used to check for network connectivity and response times.
```
Comment (lmilby-r7) [inline ping/plugin.spec.yaml]:
```suggestion
description: The Ping plugin is used to check for network connectivity and response times
```

File: port_knocking/plugin.spec.yaml [hunk 5]
```diff
@@ -6,7 +6,7 @@ title: Port Knocking
 vendor: rapid7
 support: community
 status: []
-description: Knocks the specified ports on a host, optionally including a payload
+description: The Pork Knocking plugin knocks the specified ports on a host and optionally supports a payload.
```
Comment (lmilby-r7) [inline port_knocking/plugin.spec.yaml]:
```suggestion
description: The Pork Knocking plugin knocks the specified ports on a host and optionally supports a payload
```

File: powershell/help.md [hunk 6]
```diff
@@ -1,17 +1,23 @@
...
-* Example: Requires an API Key from the product
-* Example: API must be enabled on the Settings page in the product
+The requirements for running the Powershell plugin will vary depending on the use case. Your powershell scripts can either be run locally on the InsightConnect Orchestrator or remotely on a powershell enabled server. 
+
+## Remote Requirements
+* A powershell enabled server
+* Administrative credentials to a Powershell enabled server
+
+## Local Requirements
+* Powershell can be run locally on the InsightConnect Orchestrator, in this case, you must assure connectivity to any network resources the Powershell script will use. Typically this involves opening ports in your firewall or windows servers.
```
Comment (hwilsey-r7) [inline powershell/help.md]: Powershell can be run locally on the InsightConnect Orchestrator. In this case...
assure -> ensure

File: powershell/plugin.spec.yaml [hunk 7]
```diff
@@ -3,19 +3,20 @@ extension: plugin
 product: ["insightconnect"]
 name: powershell
 title: PowerShell
-description: Run a PowerShell script on a remote host
+description: The Powershell plugin runs a PowerShell script on a remote host or locally on the InsightConnect Orchestrator.
```
Comment (lmilby-r7) [inline powershell/plugin.spec.yaml]:
```suggestion
description: The Powershell plugin runs a PowerShell script on a remote host or locally on the InsightConnect Orchestrator
```

File: proofpoint_url_defense/help.md [hunk 8]
```diff
@@ -5,14 +5,11 @@ This plugin decodes URLs that are encoded by Proofpoints URL Defense service usi
 
 # Key Features
 
-* Feature 1
-* Feature 2
-* Feature 3
+* Decode a URL to it's original form
```
Comment (hwilsey-r7) [inline proofpoint_url_defense/help.md]: it's -> its

File: pushover/plugin.spec.yaml [hunk 9]
```diff
@@ -3,7 +3,7 @@ extension: plugin
 product: ["insightconnect"]
 name: pushover
 title: Pushover.net
-description: Send push messages through Pushover.net
+description: The Pushover plugin allows you to send Pushover notifications.
```
Comment (lmilby-r7) [inline pushover/plugin.spec.yaml]:
```suggestion
description: The Pushover plugin allows you to send Pushover notifications
```

File: qradar/plugin.spec.yaml [hunk 10]
```diff
@@ -3,19 +3,20 @@ extension: plugin
 product: ["insightconnect"]
 name: qradar
 title: QRadar Security Intelligence Platform
-description: IBM QRadar is an enterprise security information and event management
+description: The QRadar plugin allows you to run Ariel queries and retrieve policy offense data.
```
Comment (lmilby-r7) [inline qradar/plugin.spec.yaml]:
```suggestion
description: The QRadar plugin allows you to run Ariel queries and retrieve policy offense data
```

File: python_3_script/help.md [hunk 11]
```diff
@@ -9,16 +9,17 @@ The Python 3 Script plugin allows you to run Python 3 code. It includes Python 3
 * [pyyaml 3.12](http://pyyaml.org/)
 * [records 0.5.2](https://github.com/kennethreitz/records)
 
+The Python 3 plugin also allows you load custom modules via its connection parameters
```
Comment (hwilsey-r7) [inline python_3_script/help.md]: you load -> you to load

File: qualys_ssl/plugin.spec.yaml [hunk 12]
```diff
@@ -3,21 +3,21 @@ extension: plugin
 product: ["insightconnect"]
 name: qualys_ssl
 title: Qualys SSL Labs
-description: Qualys SSL Labs is a collection of documents, tools and thoughts related
-  to SSL
+description: The Qualys SSL plugin allows the user to get information and test SSL servers.
```
Comment (lmilby-r7) [inline qualys_ssl/plugin.spec.yaml]:
```suggestion
description: The Qualys SSL plugin allows the user to get information and test SSL servers
```

**Pull request merged**